common/menu-entry.c

```diff
@@ -322,42 +322,33 @@ bool menuEntryLoad(menuEntry_s* me, const char* name, bool shortcut) {
 }
 
 void menuEntryParseIcon(menuEntry_s* me) {
-    uint8_t *imageptr = NULL;
     size_t imagesize = 256*256*3;
+    unsigned char imageptr[imagesize];
```

Comment:
This is 192KB on the stack. Can't you recycle the buffer you are malloc'ing below (line 346) instead of putting this on the stack?

Reply:
https://github.com/leapmotion/libjpeg-turbo/blob/master/turbojpeg.c

it doesnt copy it over... and i was trying to use a nullptr but it wont allow that.

Reply:
ohh, i get it... i implemented it rn... about to push